src/prognostic_equations/edmfx_sgs_flux.jl

```diff
@@ -76,10 +82,107 @@ function edmfx_sgs_mass_flux_tendency!(
+                vtt = vertical_transport(
+                    ᶜρʲs.:($j),
+                    ᶠu³_diff,
+                    ᶜa_scalar,
+                    dt,
+                    edmfx_sgsflux_upwinding,
+                )
+                @. Yₜ.c.ρq_liq += vtt
+
+                @. ᶜa_scalar =
+                    (Y.c.sgsʲs.:($$j).q_ice - ᶜspecific.q_ice) *
+                    draft_area(Y.c.sgsʲs.:($$j).ρa, ᶜρʲs.:($$j))
+                vtt = vertical_transport(
+                    ᶜρʲs.:($j),
+                    ᶠu³_diff,
+                    ᶜa_scalar,
+                    dt,
+                    edmfx_sgsflux_upwinding,
+                )
+                @. Yₜ.c.ρq_ice += vtt
+
+                @. ᶜa_scalar =
+                    (Y.c.sgsʲs.:($$j).q_rai - ᶜspecific.q_rai) *
+                    draft_area(Y.c.sgsʲs.:($$j).ρa, ᶜρʲs.:($$j))
+                vtt = vertical_transport(
+                    ᶜρʲs.:($j),
+                    ᶠu³_diff,
+                    ᶜa_scalar,
+                    dt,
+                    edmfx_sgsflux_upwinding,
+                )
+                @. Yₜ.c.ρq_rai += vtt
+
+                @. ᶜa_scalar =
+                    (Y.c.sgsʲs.:($$j).q_sno - ᶜspecific.q_sno) *
+                    draft_area(Y.c.sgsʲs.:($$j).ρa, ᶜρʲs.:($$j))
+                vtt = vertical_transport(
+                    ᶜρʲs.:($j),
+                    ᶠu³_diff,
+                    ᶜa_scalar,
+                    dt,
+                    edmfx_sgsflux_upwinding,
+                )
+                @. Yₜ.c.ρq_sno += vtt
+            end
+            @. ᶠu³_diff = ᶠu³⁰ - ᶠu³
+
+            @. ᶜa_scalar = (ᶜq_liq⁰ - ᶜspecific.q_liq) * draft_area(ᶜρa⁰, ᶜρ⁰)
+            vtt = vertical_transport(
+                ᶜρ⁰,
+                ᶠu³_diff,
+                ᶜa_scalar,
+                dt,
+                edmfx_sgsflux_upwinding,
+            )
+            @. Yₜ.c.ρq_liq += vtt
+
+            @. ᶜa_scalar = (ᶜq_ice⁰ - ᶜspecific.q_ice) * draft_area(ᶜρa⁰, ᶜρ⁰)
+            vtt = vertical_transport(
+                ᶜρ⁰,
+                ᶠu³_diff,
+                ᶜa_scalar,
+                dt,
+                edmfx_sgsflux_upwinding,
+            )
+            @. Yₜ.c.ρq_ice += vtt
+
+            @. ᶜa_scalar = (ᶜq_rai⁰ - ᶜspecific.q_rai) * draft_area(ᶜρa⁰, ᶜρ⁰)
+            vtt = vertical_transport(
+                ᶜρ⁰,
+                ᶠu³_diff,
+                ᶜa_scalar,
+                dt,
+                edmfx_sgsflux_upwinding,
+            )
+            @. Yₜ.c.ρq_rai += vtt
+
+            @. ᶜa_scalar = (ᶜq_sno⁰ - ᶜspecific.q_sno) * draft_area(ᶜρa⁰, ᶜρ⁰)
+            vtt = vertical_transport(
+                ᶜρ⁰,
+                ᶠu³_diff,
+                ᶜa_scalar,
+                dt,
+                edmfx_sgsflux_upwinding,
+            )
+            @. Yₜ.c.ρq_sno += vtt
+        end
+        # TODO - compute sedimentation and terminal velocities
+        # TODO - add w q_tot, w h_tot terms
```

Comment:
```suggestion
```
I don't think we need these here. But I can double check.

Reply:
I thought I would need to modify 

`ᶠu³_diff = ᶠu³ʲs.:($$j) - ᶠu³`

to something like

`ᶠu³_diff = ᶠu³ʲs.:($$j) + terminal_velocity_in_updraft($$j) - ᶠu³ + terminal_velocity_grid_mean `